internal/app/machined/pkg/controllers/k8s/nodeip_config.go

```diff
@@ -89,6 +89,9 @@ func (ctrl *NodeIPConfigController) Run(ctx context.Context, r controller.Runtim
 
 				spec.ExcludeSubnets = nil
 
+				// filer out Pod CIDRs, they can't be kubelet IPs
+				spec.ExcludeSubnets = append(spec.ExcludeSubnets, cfgProvider.Cluster().Network().PodCIDRs()...)
```

Comment:
would this exclude service IP's being broadcasted as ARP IP's? I've seen the same when the service IP's are broadcasted by kube-vip

Reply:
nope, but let me add that, safe to exclude service CIDRs as well

Reply:
the service CIDR's could be different from the ARP range though, but I don;t see any easy way to ignore this